### PR TITLE
[WIP] Aw cart styling #18

### PR DESCRIPTION
**Pull request by wlffann**
This still needs a little clean up with layout but should be good for functionality.

Do we need a way to change quantity in the cart?

### CODE REVIEW

File: app/views/carts/index.html.haml [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+    - @items.each do |item|
+      %li.cart-item
+        %section.quantity
+          = "Quantity: #{@cart.count_of(item.id)}"
+        .img-container
+          %img{ :src => "#{ item.image_url }", :alt => "item.name", :class => "img-responsive" }
+        %section.info
+          %h2= item.name
+          %p= item.description
+          %p= "#{item.price_per_unit} / lb"
```
Comment (tmikeschu) [inline app/views/carts/index.html.haml]: Probably throw a dollar sign in line 13 before the interpolation?

Comment (wlffann) [inline app/views/carts/index.html.haml]: Actually I took the quotes and it still works!

File: app/assets/stylesheets/cart.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        background-color: #a73824;
+        padding: 5px;
+        border-radius: 5px;
+      }
+    }
+  }
+  .cart_total {
+    float: right;
+  }
+  @media (min-width: 768px) {
```
Comment (tmikeschu) [inline app/assets/stylesheets/cart.scss]: Responsive! Woohoo!

**Comment by tmikeschu** [comment]
Nice work, @wlffann ! I like how you took the existing iteration structure and added all the css selectors to it.

File: app/views/carts/index.html.haml (hunk 1 above)
Comment (wlffann) [inline app/views/carts/index.html.haml]: WOAH re-read and I see what you mean. "$" added!

**Comment by akintner** [comment]
ready, I'd say!

**Pull request merged**